Client/Frontend/Browser/Tabs/TabTrayV2ViewModel.swift

```diff
@@ -39,22 +39,20 @@ class TabTrayV2ViewModel: NSObject {
         super.init()
         tabManager.addDelegate(self)
         register(self, forTabEvents: .didLoadFavicon, .didChangeURL)
-        setupPrivateModeBadge()
     }
     
     // Returns tabs for the mode the current view model is in
     func getTabs() -> [Tab] {
         return self.isPrivate ? tabManager.privateTabs : tabManager.normalTabs
     }
     
-    func setupPrivateModeBadge() {
-        viewController.toolbar.maskButton.setSelected(isPrivate, animated: true)
-        viewController.toolbar.applyUIMode(isPrivate: isPrivate)
+    func countOfNormalTabs() -> Int {
+        return tabManager.normalTabs.count
     }
 
-    func togglePrivateMode () {
+    func togglePrivateMode (_ toggleToOn: Bool) {
```

Comment:
nit: instead of `toggleToOn` it would be nice if we can be more descriptive and name it as `isPrivate` for better readability